miden/Cargo.toml

```diff
@@ -43,6 +43,7 @@ executable = ["env_logger", "hex/std", "std", "serde/std", "serde_derive", "serd
 std = ["assembly/std", "log/std", "processor/std", "prover/std", "verifier/std"]
 
 [dependencies]
+air = { package = "miden-air", path = "../air", version = "0.7", default-features = false }
```

Comment:
Are we importing this to get `ExecutionOptions`? If so, these should be re-exported from the `processor` crate.